src/component/navigation/script/navigation/navigation.js

```diff
@@ -21,7 +21,7 @@ class Navigation extends api.core.CollapsesGroup {
 
   down (e) {
     if (!this.isBreakpoint(api.core.Breakpoints.LG) || this.index === -1) return;
-    this.position = this.current.element.node.contains(e.target) ? NavigationMousePosition.INSIDE : NavigationMousePosition.OUTSIDE;
+    this.position = this.current && this.current.element.node.contains(e.target) ? NavigationMousePosition.INSIDE : NavigationMousePosition.OUTSIDE;
```

Comment:
Inclure le check de this.current dans le return
if (!this.isBreakpoint(api.core.Breakpoints.LG) || this.index === -1 || !this.current) return;

Reply:
✅